uvtools/dspec.py

```diff
@@ -477,6 +498,7 @@ def fourier_filter(x, data, wgts, filter_centers, filter_half_widths, mode,
                            filter_dims_d = [1]
                        suppression_factors = filter_kwargs.pop('suppression_factors')
                        max_contiguous_edge_flags = filter_kwargs.pop('max_contiguous_edge_flags')
+                       filt2d_mode = filter_kwargs.pop('filt2d_mode')
```

Comment:
`filt2d_mode` is popped from `filter_kwargs` here, but it doesn't look like it's passed to `_fit_basis_2d`&mdash;is this intentional?